scripts/create-kubestellar-demo-env.sh

```diff
@@ -169,6 +169,10 @@ for cluster in "${clusters[@]}"; do
     }; then
         echo -e "\033[33m✔\033[0m Cluster $cluster was successfully created"
         kubectl config rename-context "${k8s_platform}-${cluster}" "${cluster}" >/dev/null 2>&1
+        
+        # Wait for cluster to be ready before proceeding
+        echo "Waiting for cluster $cluster nodes to be ready..."
+        kubectl --context "${cluster}" wait --for=condition=ready nodes --all --timeout=300s
```

Comment:
Why is this wait here? The following command does not depend on the WEC cluster nodes being "ready".